test/Graphics/Image.cpp

```diff
@@ -0,0 +1,151 @@
+#include <SFML/Graphics/Image.hpp>
+#include "GraphicsUtil.hpp"
+#include <array>
+
+#include <doctest.h>
+
+TEST_CASE("sf::Image - [graphics]")
+{
+    SUBCASE("Construction")
+    {
+        SUBCASE("Default constructor")
+        {
+            const sf::Image image;
+            CHECK(image.getSize() == sf::Vector2u());
+            CHECK(image.getPixelsPtr() == nullptr);
+        }
+    }
```

Comment:
You don't need this redundant scope since there is only one constructor to test.